app/javascript/components/applications/application_form/PaperApplicationForm.js

```diff
@@ -59,10 +60,24 @@ const buildPrefValidations = (prefs) => {
   return prefValidations
 }
 
+const buildHouseholdMemberValidations = (householdMembers) => {
+  let householdMemberValidations = {}
+  forEach(householdMembers, (member, index) => {
+    householdMemberValidations[index] = {
+      first_name: validate.isPresent('Please enter a First Name')(member.first_name),
+      last_name: validate.isPresent('Please enter a Last Name')(member.last_name),
+      date_of_birth: validate.isValidDate('Please enter a valid Date of Birth')(member.date_of_birth)
+    }
+  })
+  return householdMemberValidations
+}
+
 const validateError = (values) => {
-  return {
-    preferences: buildPrefValidations(values.preferences)
+  const validateErrors = {
+    preferences: buildPrefValidations(values.preferences),
+    household_members: buildHouseholdMemberValidations(values.household_members)
   }
+  return validateErrors
 }
```

Comment:
trying out my first GitHub suggestion! ✨ looks like only one-line suggestions are supported at this time, so i wasn't able to select the entire `validateError` function, but hopefully you can see my suggestion is for a slight refactor to that function, not to replace just this bracket lol
```suggestion
const validateError = (values) => (
  {
    preferences: buildPrefValidations(values.preferences),
    household_members: buildHouseholdMemberValidations(values.household_members)
  }
)
```